vendor/src/github.com/seccomp/libseccomp-golang/seccomp.go

```diff
@@ -20,7 +20,7 @@ import (
 
 // C wrapping code
 
-// #cgo LDFLAGS: -lseccomp
+// #cgo pkg-config: libseccomp
```

Comment:
> Perhaps because Dockerfile.gccgo has no "seccomp" in DOCKER_BUILDTAGS?

Hmm, no it has: `ENV DOCKER_BUILDTAGS apparmor seccomp selinux`
But since 40b21745cc14628fc5336fbacddf092667a70c65 it uses "gcc 6.1".